--------------------------------------------------
test/INST_81x81/60/inst81x81_60_0.txt : 1005986 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_10.txt : 1005958 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_11.txt : 1006016 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_12.txt : 1005776 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_13.txt : 1006434 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_14.txt : 1006119 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_15.txt : 1005833 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_16.txt : 1005822 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_17.txt : 1006000 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_18.txt : 1006019 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_19.txt : 1005813 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_1.txt : 1006114 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_20.txt : 1006177 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_21.txt : 1006109 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_22.txt : 1006064 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_23.txt : 1005677 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_24.txt : 1006012 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_25.txt : 1005840 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_26.txt : 1005786 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_27.txt : 1006327 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_28.txt : 1006130 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_29.txt : 1006109 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_2.txt : 1005573 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_30.txt : 1006190 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_31.txt : 1005644 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_32.txt : 1005862 ms.
--------------------------------------------------
test/INST_81x81/60/inst81x81_60_33.txt : 1005946 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_34.txt'
test/INST_81x81/60/inst81x81_60_34.txt : 1517 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_35.txt'
test/INST_81x81/60/inst81x81_60_35.txt : 282 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_36.txt'
test/INST_81x81/60/inst81x81_60_36.txt : 276 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_37.txt'
test/INST_81x81/60/inst81x81_60_37.txt : 285 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_38.txt'
test/INST_81x81/60/inst81x81_60_38.txt : 293 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_39.txt'
test/INST_81x81/60/inst81x81_60_39.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_3.txt'
test/INST_81x81/60/inst81x81_60_3.txt : 286 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_40.txt'
test/INST_81x81/60/inst81x81_60_40.txt : 284 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_41.txt'
test/INST_81x81/60/inst81x81_60_41.txt : 236 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_42.txt'
test/INST_81x81/60/inst81x81_60_42.txt : 284 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_43.txt'
test/INST_81x81/60/inst81x81_60_43.txt : 298 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_44.txt'
test/INST_81x81/60/inst81x81_60_44.txt : 288 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_45.txt'
test/INST_81x81/60/inst81x81_60_45.txt : 298 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_46.txt'
test/INST_81x81/60/inst81x81_60_46.txt : 298 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_47.txt'
test/INST_81x81/60/inst81x81_60_47.txt : 288 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_48.txt'
test/INST_81x81/60/inst81x81_60_48.txt : 258 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_49.txt'
test/INST_81x81/60/inst81x81_60_49.txt : 301 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_4.txt'
test/INST_81x81/60/inst81x81_60_4.txt : 309 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_50.txt'
test/INST_81x81/60/inst81x81_60_50.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_51.txt'
test/INST_81x81/60/inst81x81_60_51.txt : 291 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_52.txt'
test/INST_81x81/60/inst81x81_60_52.txt : 292 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_53.txt'
test/INST_81x81/60/inst81x81_60_53.txt : 288 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_54.txt'
test/INST_81x81/60/inst81x81_60_54.txt : 293 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_55.txt'
test/INST_81x81/60/inst81x81_60_55.txt : 324 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_56.txt'
test/INST_81x81/60/inst81x81_60_56.txt : 294 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_57.txt'
test/INST_81x81/60/inst81x81_60_57.txt : 286 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_58.txt'
test/INST_81x81/60/inst81x81_60_58.txt : 271 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_59.txt'
test/INST_81x81/60/inst81x81_60_59.txt : 285 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_5.txt'
test/INST_81x81/60/inst81x81_60_5.txt : 281 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_60.txt'
test/INST_81x81/60/inst81x81_60_60.txt : 296 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_61.txt'
test/INST_81x81/60/inst81x81_60_61.txt : 285 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_62.txt'
test/INST_81x81/60/inst81x81_60_62.txt : 290 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_63.txt'
test/INST_81x81/60/inst81x81_60_63.txt : 292 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_64.txt'
test/INST_81x81/60/inst81x81_60_64.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_65.txt'
test/INST_81x81/60/inst81x81_60_65.txt : 284 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_66.txt'
test/INST_81x81/60/inst81x81_60_66.txt : 287 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_67.txt'
test/INST_81x81/60/inst81x81_60_67.txt : 293 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_68.txt'
test/INST_81x81/60/inst81x81_60_68.txt : 472 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_69.txt'
test/INST_81x81/60/inst81x81_60_69.txt : 626 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_6.txt'
test/INST_81x81/60/inst81x81_60_6.txt : 630 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_70.txt'
test/INST_81x81/60/inst81x81_60_70.txt : 539 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_71.txt'
test/INST_81x81/60/inst81x81_60_71.txt : 296 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_72.txt'
test/INST_81x81/60/inst81x81_60_72.txt : 226 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_73.txt'
test/INST_81x81/60/inst81x81_60_73.txt : 279 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_74.txt'
test/INST_81x81/60/inst81x81_60_74.txt : 291 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_75.txt'
test/INST_81x81/60/inst81x81_60_75.txt : 290 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_76.txt'
test/INST_81x81/60/inst81x81_60_76.txt : 282 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_77.txt'
test/INST_81x81/60/inst81x81_60_77.txt : 285 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_78.txt'
test/INST_81x81/60/inst81x81_60_78.txt : 280 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_79.txt'
test/INST_81x81/60/inst81x81_60_79.txt : 286 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_7.txt'
test/INST_81x81/60/inst81x81_60_7.txt : 292 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_80.txt'
test/INST_81x81/60/inst81x81_60_80.txt : 284 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_81.txt'
test/INST_81x81/60/inst81x81_60_81.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_82.txt'
test/INST_81x81/60/inst81x81_60_82.txt : 286 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_83.txt'
test/INST_81x81/60/inst81x81_60_83.txt : 287 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_84.txt'
test/INST_81x81/60/inst81x81_60_84.txt : 282 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_85.txt'
test/INST_81x81/60/inst81x81_60_85.txt : 267 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_86.txt'
test/INST_81x81/60/inst81x81_60_86.txt : 294 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_87.txt'
test/INST_81x81/60/inst81x81_60_87.txt : 283 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_88.txt'
test/INST_81x81/60/inst81x81_60_88.txt : 243 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_89.txt'
test/INST_81x81/60/inst81x81_60_89.txt : 282 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_8.txt'
test/INST_81x81/60/inst81x81_60_8.txt : 316 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_90.txt'
test/INST_81x81/60/inst81x81_60_90.txt : 291 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_91.txt'
test/INST_81x81/60/inst81x81_60_91.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_92.txt'
test/INST_81x81/60/inst81x81_60_92.txt : 289 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_93.txt'
test/INST_81x81/60/inst81x81_60_93.txt : 257 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_94.txt'
test/INST_81x81/60/inst81x81_60_94.txt : 277 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_95.txt'
test/INST_81x81/60/inst81x81_60_95.txt : 296 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_96.txt'
test/INST_81x81/60/inst81x81_60_96.txt : 306 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_97.txt'
test/INST_81x81/60/inst81x81_60_97.txt : 281 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_98.txt'
test/INST_81x81/60/inst81x81_60_98.txt : 288 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_99.txt'
test/INST_81x81/60/inst81x81_60_99.txt : 295 ms.
--------------------------------------------------
Traceback (most recent call last):
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 85, in <module>
    solve_sudoku(file)
  File "/home/hanrui/Sudoku/solvers/sudoku_ort/sudoku.py", line 33, in solve_sudoku
    with open(file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'test/INST_81x81/60/inst81x81_60_9.txt'
test/INST_81x81/60/inst81x81_60_9.txt : 230 ms.
